Chore/ChoreMenu.js

```diff
@@ -3,53 +3,101 @@
         runChoreIntroduction();
     }
 
-    sendVirtualAssistantMessage('Do you wish to do a new chore or return?', 0);
-
-    let lobbyAnswer = createInput('Chore', 'Info', 'Return');
-    while (true) {
+ 
+    setVar("doingchores",true);
+ 
+    while (getVar("doingchores")==true) {
+		   sendVirtualAssistantMessage('Do you wish to do a new chore or return?', 0);
+		   let lobbyAnswer = createInput('Chore', 'Info', 'Return');
         if (lobbyAnswer.isLike('chore', 'yes', 'new')) {
             lobbyAnswer.clearOptions();
+				sendVirtualAssistantMessage("test1");
+			//(assignedorpick)
+			doassigned=false;
+			sendVirtualAssistantMessage("test2");
+			if((isVar("AssignedChores") && getVar("AssignedChores")==true) &&(getVar("doingchores")==true)){
+				sendVirtualAssistantMessage(" Do you have a chore already chosen by %DomHonorific% %DomName% or should I PICK one for you?",0); 
+				showImage("Images/Spicy/GNMBackground/TrashBin.*");
+				canswer=createInput("Assigned by Mistress", "please pick one");
+				while (true) {
+					if (canswer.isLike("assigned", "chosen")) {
+					  sendVirtualAssistantMessage(" %subname%, you're lucky to have such an attentive %DomHonorific% in %Domname%  you %Slave% ",0);
+						showImage("Images/Spicy/FFriends/Girlfriend/*");
```

Comment:
Talking about domme but showing girlfriend?

Reply:
true... thats wonky... I'll think of some rewording